app/views/idv/in_person/verify_info/show.html.erb

```diff
@@ -160,12 +160,20 @@ locals:
           ) %>
     </div>
     <div class='grid-auto'>
-      <%= button_to(
-            idv_in_person_step_url(step: :redo_ssn),
-            method: :put,
-            class: 'usa-button usa-button--unstyled padding-y-1',
+      <% if IdentityConfig.store.in_person_ssn_info_controller_enabled %>
```

Comment:
just checking - will this be changed in the delete pr?

Reply:
@svalexander Yes! All instance of IdentityConfig.store.in_person_ssn_info_controller_enabled will need to be deleted. This is so our Update link on Verify Info takes our user to the right SSN contoller (non-FSM or FSM).

If the flag is enabled, we want to direct the user to the SSN controller based step (url: verify/in_person_proofing/ssn --- notice the in_person_proofing url, it is not in_person only- it is the new route).

If the flag is disabled, we want to direct the user to the FSM SSN step so we have to pass in the step name - (idv_in_person_step_url(step: :redo_ssn)